packages/amplify-e2e-core/src/asciinema-recorder.ts

```diff
@@ -53,6 +53,9 @@ export class Recorder {
     if (this.exitCode !== undefined) {
       throw new Error('Already executed. Please start a new instance');
     }
+    if (process.env.VERBOSE_LOGGING_DO_NOT_USE_IN_CI_OR_YOU_WILL_BE_FIRED) {
+      console.log(`Executing command [${this.cmd} ${this.args.join(' ')}]`);
+    }
```

Comment:
I was working on similar change last week. But I ended up closing - I discovered that command is dumped into cast files.
But, since you're proposing similar change then why don't we remove if and just log whole command line?

Reply:
The only thing I'm worried about is if we have a test that passes in credentials to a headless command. We'd need to be sure we don't log that

Reply:
The version I was experimenting with was to log first ~3 tokens of the command. Maybe that would work?

Otherwise, if you want to keep current version I'd change if condition to `process.env.VERBOSE_LOGGING_DO_NOT_USE_IN_CI_OR_YOU_WILL_BE_FIRED || !isCi())` so that we do log this implicitly when working on laptop.